code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```diff
@@ -532,6 +534,8 @@ func (fsh *StorageHandler) CommitWrite(ctx context.Context, r *http.Request) (*C
 	if err != nil {
 		return nil, err
 	}
+
+	fmt.Println(rootRef.Hash + ":" + strconv.FormatInt(int64(writeMarker.Timestamp), 10))
```

Comment:
Why are your using `fmt.Println` rather than a logger? It its just a debug line please remove.